### ISSUE TITLE
Missing "module" script path in Custom Elements output target

### ISSUE THREAD

**Issue by lodybo** [ionic-team/stencil-site#885]
The documentation page for the custom elements bundle is missing an example for the package.json `"module"` property.

On [line 102](https://github.com/ionic-team/stencil-site/blob/master/src/docs/output-targets/custom-elements.md?plain=1#L102) you can see that there is an instruction to set the `"module"` property, but the actual example it points to is missing.

In the example below that, the one specifying `@stencil/core` to be a dependency, we can see the `module` property being set:

```tsx
{
  "module": "dist/components/index.js",
  "dependencies": {
    "@stencil/core": "latest"
  },
  ...
}
```

Is it assumable that this is the path meant to be added in the missing instruction?

**Comment by rwaskiewicz** [comment]
@lodybo Yes, you're right. The example there is meant to cover both statements (about setting the `module` property and setting `@stencil/core` as a dependency). However, it doesn't read well/looks confusing as is today.
